components/crypto/Cargo.toml

```diff
@@ -10,6 +10,7 @@ license = "Apache-2.0"
 keywords = ["exonum", "cryptography", "library"]
 categories = ["cryptography", "hashing"]
 description = "Cryptography related types, constants, traits and functions."
+links = "exonum_protobuf_crypto"
```

Comment:
What happens if the `protobuf` function is disabled.?

Reply:
I've tested it, nothing will happend.